torch/special/__init__.py

```diff
@@ -505,6 +505,32 @@
 Alias for :func:`torch.round`.
 """)
 
+log_softmax = _add_docstr(_special.special_log_softmax,
+                          r"""
+Computes softmax followed by a logarithm.
+
+While mathematically equivalent to log(softmax(x)), doing these two
+operations separately is slower, and numerically unstable. This function
```

Comment:
no comma needed on this line

Reply:
Have also updated the original docs from `nn.functional`.